README.md

```diff
@@ -342,17 +372,17 @@ fully functional for all operating systems, including Windows and macOS.
 -   `archlinux` (Arch Linux)
 -   `arcolinux` (Arco Linux)
 -   `artixlinux` (Artix Linux)
--   `athenaos` (Athenaos)
+-   `athenaos` (Athena OS)
```

Comment:
These changed arise from `cog` : presumably the revised approach to pretty_name needs some additional data or a tweak